src/cache.rs

```diff
@@ -61,17 +63,29 @@ impl Cache {
     pub fn get_cache_dir() -> Result<(PathBuf, PathSource), TealdeerError> {
         // Allow overriding the cache directory by setting the
         // $TEALDEER_CACHE_DIR env variable.
```

Comment:
Since the variable name is now a constant, we should remove other occurences:

```suggestion
        // env variable.
```

I am not sure if it appears in other places too

Reply:
It's only this one occurrence - removed it.